Source/SelfService/Web/logging/logFilter/filterSelect.tsx

```diff
@@ -2,8 +2,35 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 import React from 'react';
+
+import { themeDark } from '../../theme/theme';
 import { Select, SelectProps } from '@mui/material';
 
+const styles = {
+    'padding': '5px 14px',
+    'borderRadius': '4px',
```

Comment:
Since these spacings effect the external paddings of the component, I think it would make sense to lean on MUI-spacing. The benefit of this is that if we ever change the fundamental look&feel of the theme, then the spacing will stay relative.

Docker has this in their MUI guidelines as well, which I think makes sense to lean on: https://docs.docker.com/desktop/extensions-sdk/design/mui-best-practices/

Reply:
the default spacing value (which is what we are using) is 1 = 8px, so 5px is 0.625, and 14px is 1.75. 

You can use the spacing variants, or a spacing util to specify multiple values ex: `padding: theme.spacing(0.625, 1.75)` or `px: 0.625, py: 1.75`.

More info and variants here: https://mui.com/material-ui/customization/spacing/#multiple-arity